src/features/swap/components/SwapOrderConfirmationView/index.tsx

```diff
@@ -107,8 +117,32 @@ export const SwapOrderConfirmationView = ({ order, settlementContract }: SwapOrd
           ),
         ]}
       />
+
+      {isTwapOrder && (
+        <div className={css.partsBlock}>
+          <DataTable
+            rows={[
+              <Typography key="title" variant="body1" className={css.partsBlockTitle}>
+                <strong>
+                  Order will be split in{' '}
+                  <span className={css.numberOfPartsLabel}>${order.numberOfParts} equal parts</span>
+                </strong>
+              </Typography>,
+              <PartSellAmount order={order} addonText="per part" key="sell_part" />,
+              <PartBuyAmount order={order} addonText="per part" key="buy_part" />,
+              <DataRow title="Start time" key="Start time">
+                {order.startTime.startType === StartTimeValue.AtMiningTime ? 'At mining time' : 'At block number'}
```

Comment:
Not sure how useful this information is for the user. In the CoW review screen it just says `Start time first part: Now` which is easier to understand imo.

Reply:
As I wrote on the designs - we need to come with something better than "now", because "now" in my view means nothing. It can be now, it can be tomorrow - it is at the moment this tx gets minted. This needs to be clear to the user.

Reply:
Imo `Now` is easy to understand because it means as soon as the transaction is executed whereas differentiating between mining and block number is quite technical. Where is the difference between those two?

Reply:
It cannot be "now" if the transaction is not submitted, right? If I have n/m safe it's not now. It's a point of time in the future :)